src/player.cpp

```diff
@@ -10215,7 +10215,7 @@ bool player::wear_item(item *to_wear, bool interactive)
         }
 
         // this simply checked if it was zero, I've updated this for the new encumb system
-        if (to_wear->covers(bp_head) && (encumb(bp_head) + to_wear->get_encumber()) > 20) {
+        if (to_wear->covers(bp_head) && (encumb(bp_head) > 10 || ((to_wear->get_encumber()) < 10 && to_wear->has_flag("FIT")))) {
```

Comment:
I'd feel a little more comfortable with the OR'd component in parens.  Probably not necessary, but wev.